contracts/VeJoeStaking.sol

```diff
@@ -135,15 +135,15 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
 
         require(_speedUpDuration <= 365 days, "VeJoeStaking: expected _speedUpDuration to be <= 365 days");
 
-        upperLimitMaxCap = 100000;
+        upperLimitMaxCapPct = 10000000;
         require(
-            _maxCap != 0 && _maxCap <= upperLimitMaxCap,
-            "VeJoeStaking: expected _maxCap to be non-zero and <= 100000"
+            _maxCapPct != 0 && _maxCapPct <= upperLimitMaxCapPct,
+            "VeJoeStaking: expected _maxCapPct to be non-zero and <= 10000000"
         );
 
         __Ownable_init();
```

Comment:
Inits should always be the first line of code of initializer.
Cause during the constructor we could execute code and `owner` wouldn't be set yet